builder/amazon/common/step_key_pair.go

```diff
@@ -36,7 +37,7 @@ func (s *StepKeyPair) Run(_ context.Context, state multistep.StateBag) multistep
 		}
 
 		state.Put("keyPair", s.KeyPairName)
-		state.Put("privateKey", string(privateKeyBytes))
+		state.Put("privateKey", strings.TrimSpace(string(privateKeyBytes)))
```

Comment:
is this change supposed to be here?

Reply:
I must have been doing some testing of a different issue on this branch. removed it.